pyenvbuilder/commands/check.py

```diff
@@ -12,12 +13,15 @@
 schema_dir = Path(__file__).absolute().parents[0] / 'schema.yml'
 
 
-def check(args):
+def check(**kwargs):
     '''
     Validates if proper files or directories were passed in
     '''
-    for arg in args:
-        arg_path = Path(arg)
+    # get the 'files' from the kwargs 'check' command
+    get_files = (v for k, v in kwargs.items() if k == 'files')
```

Comment:
`files` will show only once since it is a dictionary key.
```suggestion
    get_files = kwargs.get('files', [])
```